src/hbbft.erl

```diff
@@ -200,7 +200,7 @@ handle_msg(Data = #hbbft_data{round=R}, J, {{acs, R}, ACSMsg}) ->
                                         SerializedShare = hbbft_utils:share_to_binary(Share),
                                         {multicast, {dec, Data#hbbft_data.round, I, SerializedShare}}
                                 end, Results),
-            {Data#hbbft_data{acs=NewACS, acs_results=Results}, {send, Replies}};
+            {Data#hbbft_data{acs=NewACS, acs_results=Results}, {send,  hbbft_utils:wrap({acs, Data#hbbft_data.round}, ACSResponse) ++ Replies}};
```

Comment:
Just so I know, appending the `Replies` at the end of the list is intentional..?. Just wanna make sure that something weird wouldn't happen with the ordering of the messages.

Reply:
They're 2 ordered lists, with the 'oldest' ones first, so the total ordering should be preserved?